chain/src/chain.rs

```diff
@@ -1484,32 +1484,53 @@ impl Chain {
 	/// Migrate our local db from v2 to v3.
 	/// "commit only" inputs.
 	fn migrate_db_v2_v3(store: &ChainStore) -> Result<(), Error> {
+		if store.batch()?.is_blocks_v3_migrated()? {
```

Comment:
Just merged #3513 so rebasing this PR should clear this lot up.

Reply:
Done